package.json

```diff
@@ -28,8 +30,10 @@
   },
   "homepage": "https://github.com/devongovett/exif-reader",
   "devDependencies": {
+    "@types/node": "^16.18.31",
```

Comment:
Can we get away with using `*` for the version here (if only to avoid future possible maintenance overhead of having to update this)?

Reply:
Yes, I agree. I was thinking of a minimal version like `>4.x`, but it seems that even the lowest 0.10 version of node has a matching Buffer type.